src/onnxruntime.cc

```diff
@@ -596,6 +607,8 @@ class ModelInstanceState : public BackendModelInstance {
   // instance.
   OrtSession* session_;
   OrtAllocator* allocator_;
+  OrtIoBinding* io_binding_;
```

Comment:
nit: may be at some point we should convert these to unique_ptrs so that we don't forget to Release() them.

Reply:
Will approach this in a separate PR